cpp/examples/CMakeLists.txt

```diff
@@ -22,6 +22,22 @@ add_executable(ode_sir_example ode_sir.cpp)
 target_link_libraries(ode_sir_example PRIVATE memilio ode_sir)
 
 
+add_executable(seair_example seair.cpp)
+target_link_libraries(seair_example PRIVATE memilio ode_seair AD::AD)
+
+if(NOT WIN32 AND MEMILIO_USE_BUNDLED_IPOPT)

```

Comment:
Could you add a comment why 'NOT WIN32' is necessary

Reply:
Actually, because the Windows CI machines do not really support Fortran. Actually it works one Windows with Visual Studio 2019 if the free Intel-Fortran-Compiler ist installed. Is there a proper way to test for the CI-Machines in CMake?

Reply:
@dabele Any idea how to resolve this?

Reply:
I added the option MEMILIO_ENABLE_OPTIMIZATION (default OFF) as suggested. I remote the option USE_BUNDLED_IPOPT, since the code is not tested and will not work with system Ipopt. 

I check for MEMILIO_ENABLE_OPTIMIZATION to conditionally include the optimization example.